app/services/proceeding_type_service.rb

```diff
@@ -55,6 +55,8 @@ def add_cost_limitations_to_response
   def add_scope_limitations_to_response
     scl = substantive_scope_limitations
     dfcl = delegated_functions_scope_limitations
+    return unless scl && dfcl
```

Comment:
I know this was already here, but I think we should avoid abbreviations like this.

I'm not sure what they add, but they make the code harder to understand in my opinion.

What do you think?